robotiq_force_torque_sensor/urdf/ur5_fts_test.urdf.xacro

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="robotiq_fts150_test">
+
+    <!-- includes -->
+    <xacro:include filename="$(find robotiq_force_torque_sensor)/urdf/robotiq_fts150.urdf.xacro" />
+    <xacro:include filename="$(find ur_description)/urdf/ur5_robot.urdf.xacro" />
+
+    <!-- include universal_robot arm -->
+    <xacro:ur5 prefix=""/>
+
+    <!-- include fts -->
+    <xacro:robotiq_fts150 prefix="" parent="ee_link">
+        <origin xyz="0.009 0 0" rpy="0 ${pi/2} 0"/> <!-- This offset is the flange adapter plate -->
+    </xacro:robotiq_fts150>
+
+</robot>
```

Comment:
See earlier comment about dependency on `ur_description`.

A test / model inspection launch file is a nice addition, but perhaps it should not introduce dependencies on external packages from unrelated repositories.